sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Count.java

```diff
@@ -74,21 +74,9 @@ private Count() {
    * Returns a {@link PerElement Count.PerElement} {@link PTransform} that counts the number of
```

Comment:
Mind also cleaning up this reference? It now refers to a private class and is useless.